tests/unit/amo/components/TestEditableCollectionAddon.js

```diff
@@ -76,4 +83,164 @@ describe(__filename, () => {
     sinon.assert.called(clickEvent.stopPropagation);
     sinon.assert.calledWith(removeAddon, addon.id);
   });
+
+  describe('notes area', () => {
+    it('hides the notes area by default', () => {
+      const root = render();
+
+      expect(root.find('.EditableCollectionAddon-notes')).toHaveLength(0);
+    });
+
+    it('shows the read-only version of the notes area if there are notes', () => {
+      const notes = 'Some notes.';
+      const addon = {
+        ...fakeAddon,
+        notes,
+      };
+      const root = render({ addon });
+      const notesArea = root.find('.EditableCollectionAddon-notes');
+      expect(notesArea.find(Icon)).toHaveProp('name', 'comments-blue');
+      expect(root.find('.EditableCollectionAddon-notes-read-only'))
+        .toHaveLength(1);
+      expect(notesArea.find('.EditableCollectionAddon-notes-content'))
+        .toHaveProp('dangerouslySetInnerHTML', sanitizeHTML(notes));
```

Comment:
We usually use `toHaveHTML` so that we don't call `sanitizeHTML` here and we can assert the content in order to avoid issues such as #5230.